jmeter-java-dsl/src/test/java/us/abstracta/jmeter/javadsl/http/DslCookieManagerTest.java

```diff
@@ -41,6 +41,22 @@ public DslTestPlan testPlanWithCookiesAndNoHttpRequests() {
       );
     }
 
+    public DslTestPlan testPlanWithCookiePolicy() {
```

Comment:
For "functional" tests, you might want to add new tests to DslHttpSamplerTest, or maybe even better, move existing tests on that class to this class and add the new tests here as well.